sound/soc/amd/acp/Kconfig

```diff
@@ -126,4 +129,21 @@ config SND_SOC_AMD_SOF_MACH
 	help
 	  This option enables SOF sound card support for ACP audio.
 
+config SND_SOC_AMD_SOF_SDW_MACH
+	tristate "AMD SOF Soundwire Machine Driver Support"
+	depends on X86 && PCI && ACPI
+	depends on SOUNDWIRE
+	select SND_SOC_DMIC
+	select SND_SOC_RT711_SDW
+	select SND_SOC_RT711_SDCA_SDW
+	select SND_SOC_RT1316_SDW
+	select SND_SOC_RT715_SDW
```

Comment:
do you actually support non-sdca versions of RT711 and RT715? Those start to be ancient and not used since 2020-ish?

Reply:
Till now we have validated SDCA version codecs only.